app/players/elves/elf.py

```diff
@@ -0,0 +1,16 @@
+from abc import ABC
+from app.players.player import Player
+
+
+class Elf(Player, ABC):
```

Comment:
You don’t need to inherit from ABC as your class does not realise any abstract methods